vxquery-core/src/main/java/org/apache/vxquery/xmlquery/query/XMLQueryCompiler.java

```diff
@@ -164,13 +164,7 @@ public Object getType(ILogicalExpression expr, IMetadataProvider<?, ?> metadataP
         builder.setNullableTypeComputer(new INullableTypeComputer() {
             @Override
             public Object makeNullableType(Object type) throws AlgebricksException {
-                SequenceType st = (SequenceType) type;
-                if (st.getQuantifier().allowsEmptySequence()) {
-                    return type;
-                } else if (st.getQuantifier() == Quantifier.QUANT_ONE) {
-                    return SequenceType.create(st.getItemType(), Quantifier.QUANT_QUESTION);
-                }
-                return type;
+                throw new NotImplementedException("NullableTypeComputer is not implented (makeNullableType)");
```

Comment:
Do not remove this from master. This change is needed by the inclusion of new rewrite rules.